lang/main-fr.json

```diff
@@ -697,6 +697,8 @@
         "hangup": "Quitter",
         "help": "Aide",
         "invite": "Inviter des participants",
+        "lobbyButtonDisable": "Désactiver le contrôle des participant·e·s",
```

Comment:
I don't speak french, but shouldn't this be "participants", consistent with the line above?

Reply:
It's gender-neutral (_écriture inclusive_ in French) :)

Reply:
👍